### PR TITLE
add searchIndicators docs in demistomock

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/27068

## Description
changes in this file will be reflected in xsoar.pan.dev by the site build mechanism(within a day).
for whoever is interested, can be explained here: https://github.com/demisto/content-docs/blob/33a9c51897355e8c845a689c885d3b938a8c5afe/content-repo/create-docs.sh#L175

## Must have
- [x] Documentation

### CODE REVIEW

**Comment by tschanfeld** [comment]
@idobloch @anara123 doc review completed, minor edit made

File: Tests/demistomock/demistomock.py [hunk 1]
```diff
@@ -1022,6 +1024,26 @@ def searchIndicators(fromDate='', query='', size=100, page=0, toDate='', value='
...
+    >>> demisto.executeCommand(searchIndicators, "query": 'type:IP', "size" :1000})
+    {
+        "iocs": [],
+        "searchAfter": [1596106239679, dd7aa6abfcb3adf793922618005b2ad5],
+        "total": 7
+    }
+    ```
+    You can then use the returned value of searchAfter to iterate over all batches.
+    ```
+    >>> res = demisto.executeCommand("searchIndicators", {"query" : 'type:IP', "size" : 3})
```
Comment (idobloch) [inline Tests/demistomock/demistomock.py:1040]: probably my bad, maybe we should set the size here to the default value (50) or another number we recommend (just not leave it as 3).

**Review by idobloch — APPROVED** [review]
looks great, just a small comment about the size from my side.

File: Tests/demistomock/demistomock.py [hunk 2]
```diff
@@ -1007,7 +1007,9 @@ def createIndicators(indicators_batch, noUpdate=False):
 
 def searchIndicators(fromDate='', query='', size=100, page=0, toDate='', value='', searchAfter=None,
                      populateFields=None):
-    """Searches for indicators according to given query
+    """Searches for indicators according to given query.
+    If using Elasticsearch with Cortex XSOAR 6.1 or later,
+    the searchAfter argument must be used instead of the page argument.
```
Comment (DeanArbel) [inline Tests/demistomock/demistomock.py:1012]:
```suggestion
    the searchAfter argument will be used instead of the page argument.
```

**Review by DeanArbel — APPROVED** [review]
very nice

**Pull request merged**